src/index.js

```diff
@@ -122,12 +118,11 @@ const result = [
 	'================================ Deploy request ================================',
 	'',
 	`#devops please deploy #${packageName} ${newVersion} to production`,
-	ccList.join(' '),
-	'cc: <everyone else that should know about this change>',
+	'@@devops',
+	`CODEOWNERS: ${codeowners.join(' ')}`,
 	'',
 	'================================ Release notes =================================',
 	'',
-	`#release-notes #${packageName}`,
 	`# ${toTitleCase(packageName)} release notes ${date}`,
```

Comment:
Would this send the release email? I think we might still need the line above.

Reply:
For future reference, the discussion about this moved to FD.
See: https://www.flowdock.com/app/rulemotion/resin-devops/threads/k6V99N7enuSAct3gFoDWrmgRVed

In short, the new process says that the `#release-notes` will be used after the deploy takes place (I guess by the devops who did the deploy).
See: https://github.com/balena-io/process/pull/224/files#diff-0336c0cb761cdd1c428f55c44b0bba99R59-R60

Reply:
Make sense, thanks